eventdata/elasticlogs-index.json

```diff
@@ -12,6 +12,9 @@
       "@timestamp": {
         "type": "date"
       },
+      "_raw_event_size": {
+        "type": "short"
```

Comment:
Are we sure `raw_event` won't ever exceed 32767 characters? Looking at the [message sample](https://github.com/elastic/rally-tracks/tree/master/eventdata#example-document) seems unlikely but not sure how large the `request` string can get.

Reply:
In my tests I saw event sizes up to ~ 1000 bytes so `short` should suffice.